### DISCUSSION TITLE
Welcome to GoHeishaMon Discussions!

### DISCUSSION THREAD

**Discussion by rondoval** [rondoval/GoHeishaMon#3] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by MiG-41** [comment]
Hi , could you join to slack channel for Heishamon ? Top of ReadMe.md : https://github.com/Egyras/HeishaMon
There is also subchannel for Cz-taw1...

**Comment by MiG-41** [comment]
So what are your's next plans ?
